src/main.cpp

```diff
@@ -202,8 +202,20 @@ main(int argc, char *argv[])
     QCommandLineParser parser;
     parser.addHelpOption();
     parser.addVersionOption();
-    QCommandLineOption debugOption(QStringLiteral("debug"), QStringLiteral("Enable debug output"));
+    QCommandLineOption logLevel(
+      QStringLiteral("loglevel"),
```

Comment:
Nitpick, but can we name this "log-level"? At least if we go with "log-output", then it would at least be symmetric.

Reply:
I like symmetry. I'm reluctant to add dashes in the middle of either option, though, because they make typing awkward. For symmetry's sake, I'll leave this one alone until we reach a decision on the other one.